code/controllers/subsystem/SSeconomy.dm

```diff
@@ -285,6 +293,20 @@ SUBSYSTEM_DEF(economy)
 				objective.payout_given = TRUE
 			break
 
+/datum/controller/subsystem/economy/proc/mail_never_fails()
```

Comment:
This should probably be run on the cargo's shuttle subsystem, instead of the economy one.